tests/aws.sh

```diff
@@ -125,50 +87,49 @@ INSTANCE_ID=$(aws ec2 run-instances \
 	)
 echo INSTANCE_ID=$INSTANCE_ID
 
-aws ec2 create-tags --resources $INSTANCE_ID \
```

Comment:
could we still add some tags? So if the script is interrupted, we know what were the EC2 instances created for.